### DISCUSSION TITLE
Touchscreen leaderboard

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#27772] [Ideas] [answered]
I think touchscreen leaderboard should be added for STD similar way 4K / 7K exist in mania
There are 2 reasons for this:
- it's very difficult for touchscreen players to compete with classic players due to pp system not being able to correctly measure touchscreen difficulty
- if I want to start playing with touchscreen (or just on my phone) - it's just impossible to see your progress unless you will create multiaccount (what is not allowed)

**Comment by Rekunan** [comment, marked as answer]
- Future work in progress, eventually the pp system will be able to award accurately and properly
- Playlists are designed for this, to make whatever custom leaderboards/tracking you want.
	- You can also use a TD mod filter.

**Reply by Givikap120** [reply]
playlists and mod filters won't change the fact that i want to see my TD top plays and see my TD rank growing

**Reply by Rekunan** [reply]
Is it because of how the system works? Unfort it's too big of an ask to store a whole separate leaderboard for something like this, but maybe playlists could be reworked to make it better for you.
